src/runtime/GlobalObjectBuiltinObject.cpp

```diff
@@ -620,6 +677,11 @@ void GlobalObject::installObject(ExecutionState& state)
     m_object->defineOwnProperty(state, ObjectPropertyName(strings.freeze),
                                 ObjectPropertyDescriptor(m_objectFreeze,
                                                          (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::WritablePresent | ObjectPropertyDescriptor::ConfigurablePresent)));
+    // 19.1.2.5 Object.freeze ( O )
+    m_objectFromEntries = new NativeFunctionObject(state, NativeFunctionInfo(strings.fromEntries, builtinObjectFromEntries, 1, NativeFunctionInfo::Strict));
+    m_object->defineOwnProperty(state, ObjectPropertyName(strings.fromEntries),
+                                ObjectPropertyDescriptor(m_objectFromEntries,
+                                                         (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::WritablePresent | ObjectPropertyDescriptor::ConfigurablePresent)));
```

Comment:
How about merging these two statements by push the new statement into the argument of the defineOwnProperty?